subprojects/logging/src/integTest/groovy/org/gradle/internal/logging/ConsoleCompositeBuildFunctionalTest.groovy

```diff
@@ -17,18 +17,18 @@
 package org.gradle.internal.logging
 
 import org.gradle.integtests.fixtures.AbstractConsoleFunctionalSpec
+import org.gradle.internal.SystemProperties
 import spock.lang.Ignore
 
-@Ignore("Build fails and passes depending on timing")
+//@Ignore("Build fails and passes depending on timing")
```

Comment:
I've seen it fail before on CI. That's why I added the ignore annotation. Maybe we could try again as we made additional changes to the parsing of grouped output.